modules/auxiliary/gather/advantech_webaccess_creds.rb

```diff
@@ -0,0 +1,173 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Advantech WebAccess 8.1 Post Authentication Credential Collector",
+      'Description'    => %q{
+        This module allows you to log into Advantech WebAccess 8.1, and collect all the credentials.
+        Although authentication is required, you do not need to be admin to be able to see other
```

Comment:
this is kind of wordy, maybe something more like:
Although authentication is required, any level of user permission can exploit this vulnerability.